audeer/core/io.py

```diff
@@ -278,17 +373,25 @@ def extract_archives(
         keep_archive: bool = True,
         verbose: bool = False,
 ) -> typing.List[str]:
-    r"""Extract ZIP or TAR.GZ archives.
+    r"""Extract multiple ZIP or TAR.GZ archives at once.
 
     Args:
         archives: paths of ZIP or TAR.GZ files
-        destination: folder where the content should be stored.
-            Will be created if it doesn't exist
+        destination: folder where the files will be extracted.
+            If the folder does not exists,
+            it will be created
         keep_archive: if ``False`` delete archive files after extraction
         verbose: if ``True`` a progress bar is shown
 
     Returns:
-        combined member filenames of archives
+        extracted members
```

Comment:
Yes, makes sense to me. Maybe you can just replace `m.name` by `os.path.normpath(m.name)`.

Reply:
Maybe it's also worse adding an `if`-clause to change the path only when we running Windows as it might slow the process down if you have millions of files in the archive.

Reply:
> Maybe you can just replace m.name by os.path.normpath(m.name).

I think it's enough to do `m.name.replace('/', os.path.sep)` as we call `audeer.path()` on the files before adding them to the zip and that includes `normpath()`, right?

> Maybe it's also worth adding an if-clause to change the path only when we running Windows as it might slow the process down if you have millions of files in the archive.

Good idea.

Reply:
>> Maybe you can just replace m.name by os.path.normpath(m.name).
>
>I think it's enough to do m.name.replace('/', os.path.sep) as we call audeer.path() on the files before adding them >to the zip and that includes normpath(), right?

Yes, `m.name.replace('/', os.path.sep)` is sufficient.

Reply:
done, and changed to:

![image](https://user-images.githubusercontent.com/10383417/235119050-9f4ef01a-25ce-4a36-be46-4c67cd58eb33.png)